packages/account/src/Styles/account.scss

```diff
@@ -643,6 +649,86 @@ $MIN_HEIGHT_FLOATING: calc(
             padding-left: 8px;
         }
     }
+    .login-history-container {
+        height: 100%;
+        width: 100%;
+
+        .login-history-table {
+            @include desktop {
+                height: 100%;
+            }
+
+            @include mobile {
+                width: 90%;
+                border-collapse: separate;
+                border-spacing: 0 1em;
+                margin: 10px auto;
+                padding: 10px;
+                color: var(--text-general);
+                background-color: $color-grey-4;
+
+                &.theme-dark {
+                    background-color: $color-black-3;
```

Comment:
Please used themed colors. 🙏

Reply:
@zaki-binary The colors (`black-3` and `grey-4`) are not in the same pallet of our themed colors.
Should I create one?

Reply:
Yes please, u may create a new color palette and have it as a themed css variable constant.